M2/Macaulay2/e/todo-ncalgebra.m2

```diff
@@ -0,0 +1,242 @@
+19 May 2020  -- what needs to be done?
```

Comment:
The changelog here was very useful to read for me when I was trying to figure out this branch, but perhaps it should move to `NCAlgebras/README.md` or somewhere like that.